resource_obmcs_objectstorage_bucket.go

```diff
@@ -107,6 +114,12 @@ func (s *BucketResourceCrud) Update() (e error) {
 		opts.Metadata = metadata
 	}
 
+	if accessType, ok := s.D.GetOk("accessType"); ok {
```

Comment:
meh, we really should extract this common data binding code from Create() & Update() someday.